conf/config-default.yaml

```diff
@@ -187,3 +187,8 @@ plugins:                          # plugin list
 
 stream_plugins:
   - mqtt-proxy
```

Comment:
no plugin for `log-rotate` in list?

Reply:
when put  `log-rotate` in plugin list, it will be enabled. we don't enable it by default.

Reply:
that is right